### ISSUE TITLE
ClientReturnInfo error on LoggedOut.cshtml

### ISSUE THREAD

**Issue by ericgreenmix** [IdentityServer/IdentityServer4#124]
When logging out and hitting the `LoggedOut.cshtml` page, I am getting the following errors:

```
One or more compilation failures occurred:
/UI/Logout/Views/LoggedOut.cshtml(10,36): error CS7069: Reference to type 'ClientReturnInfo' claims it is defined in 'IdentityServer4', but it could not be found
/UI/Logout/Views/LoggedOut.cshtml(9,67): error CS7069: Reference to type 'ClientReturnInfo' claims it is defined in 'IdentityServer4', but it could not be found
```

**Comment by brockallen** [comment]
I just tried it with the latest code on dev for core and the clients and I don't see the error.

**Comment by ericgreenmix** [comment]
Okay, I got it working. It turned out to be a strange issue with the compiler and namespaces. I had renamed `Host.UI` to a new namespace that had `IdentityServer4` as part of the namespace, I.E. `MyProject.IdentityServer4.UI`. For some reason the compiler was looking in the `MyProject.IdentityServer4` assembly for the `ClientReturnInfo` type instead of the `IdentityServer4` assembly. Changing the namespace and getting rid of `IdentityServer4` in it fixed the issue.
